jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/info/JvmThreadInfo.java

```diff
@@ -0,0 +1,98 @@
+/*[INCLUDE-IF Sidecar18-SE]*/
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package openj9.tools.attach.diagnostics.info;
+import static openj9.tools.attach.diagnostics.info.ThreadGroupInfo.THREAD_GROUP_PREFIX;
+
+import java.io.IOException;
+import java.lang.management.ThreadInfo;
+import java.util.Properties;
+
+import javax.management.openmbean.CompositeData;
+import javax.management.openmbean.CompositeType;
+
+import com.ibm.java.lang.management.internal.ThreadInfoUtil;
+
+import openj9.tools.attach.diagnostics.base.DiagnosticProperties;
+
+/**
+ * Container for information on a foreign JVM's thread
+ *
+ */
+public class JvmThreadInfo  extends JvmInfo {
+
+	private static final String THREAD_INFO = THREAD_GROUP_PREFIX + "thread_info."; //$NON-NLS-1$
```

Comment:
There is only the one use of "thread_info." (for THREAD_NUM), is it really needed?

Reply:
Not really.  Deleted it and THREAD_NUM.